server/views/pages/index.njk

```diff
@@ -2,6 +2,7 @@
 {%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
```

Comment:
Think this can be removed now that we aren't using an Inset

Reply:
Oh yeah forgot about that! I'll remove 🙂